pinot-timeseries/pinot-timeseries-planner/src/main/java/org/apache/pinot/tsdb/planner/TimeSeriesQueryEnvironment.java

```diff
@@ -75,7 +76,8 @@ public void init(PinotConfiguration config) {
         throw new RuntimeException("Failed to instantiate logical planner for language: " + language, e);
       }
     }
-    TableScanVisitor.INSTANCE.init(_routingManager);
+    // TODO: Add support for logical tables in the future.
```

Comment:
nit: rename todo to `TODO(timeseries)` so we can grep and track it easily in the future